src/libsvg/path.cc

```diff
@@ -40,6 +40,7 @@
 #include <boost/tokenizer.hpp>
 #include <boost/algorithm/string.hpp>
 
+#include "core/CurveDiscretizer.h"
```

Comment:
Let's bring in @t-paul to comment on this in the context of libsvg not being dependent on OpenSCAD itself.